app/assets/stylesheets/markets.scss

```diff
@@ -0,0 +1,71 @@
+body {
+  background: url(/assets/Destruction.jpg);
+  background-size: cover;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-color: rgba(0, 0, 0, 1);
+}
+
+h6 {
+  color: #B6CA8C;
+  background-color: rgba(0, 0, 0, -100);
+  display: inline;
+}
+
+.eve-time li {
+  background: transparent;
+}
+
+#footer ul{
+  list-style-type: none;
+}
+
+#video {
+  margin-top: 150px;
+}
+
+#notice_wrapper {
+ position: absolute;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered background-color, position, top, width, z-index